web/src/beta/components/InsertionButton/index.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { action } from "@storybook/addon-actions";
+import { Meta, StoryObj } from "@storybook/react";
+
+import InsertionButton from ".";
+
+export default {
+  component: InsertionButton,
+} as Meta;
+
+type Story = StoryObj<typeof InsertionButton>;
+
+export const Default: Story = {
+  args: {
+    onClick: action("onClick"),
```

Comment:
you don't need manually pass this function.
If the field name matches [this Regex](https://github.com/reearth/reearth/blob/main/web/.storybook/preview.tsx#L39), it will be automatically attached.